src/schema/README.md

```diff
@@ -7,70 +7,382 @@ Currently the portions of the specification that rely on this schema are
 the entity tables, entity definitions, filename templates, and metadata tables.
 Any changes to the specification should be mirrored in the schema.
 
```

Comment:
This file is getting long enough that a ToC might be warranted.

Reply:
Thought about it. Would be its own thing to maintain. Note that GitHub has something built-in when viewing the file, though sadly not the diff:

![image](https://user-images.githubusercontent.com/83442/191313154-271f88a9-7b14-4552-bdb9-d50636cebac4.png)

Reply:
I'm waffling on whether the very specific sections of each of the top level namespaces should be split into their own README in that subdirectory.

Reply:
Not a bad idea. Would rather not do it right now.